.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,48 @@
+name: Pull Request Verification
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: "3.8"
+
+      - name: Setup Linux
+        if: runner.os == 'linux'
+        run: |
+          sudo apt-get install -y cmake libsndfile1
+
+      - name: Install Ludwig
+        run: pip install ludwig[full]
```

Comment:
Hey @justinxzhao , do you think we can skip this step?

Reply:
I believe we need to keep this in otherwise there's no way to get the ludwig version number.